### DISCUSSION TITLE
Красный перец. Фортуно. Лабиринты Духа.

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2nr_unofficial_update#389] [General]
Напоминаю что красный перец нужен для напитки который даёт опыт. в Готике 2 НВ, красный перец выпадал из пиратов. Димус это отключил ты хотела куда нибудь добавить его в продажу

**Comment by dosinabox** [comment]
Нужны расчеты. Количество пиратов (GIL_PIR), деленное на 6 (вероятность появления перца). А также перечень торговцев и глав, когда у них будут появляться в продаже перцы.

**Comment by Kor-Angar** [comment]
в этом списке два трупа пирата есть?
![image](https://user-images.githubusercontent.com/49514764/64210922-3972ce00-cead-11e9-8407-2843f69957de.png)

**Comment by dosinabox** [comment]
Нет. Мертвые пираты это Ангус (PIR_1370_Addon_Angus) и Хэнк (PIR_1371_Addon_Hank).
И вообще, это перечень диалогов, а не пиратов. Учти, что PIR_1389_Addon_Tom - неиспользуемый пират, который не вставляется в мир. Также нужно учесть, что некоторые пираты являются бессмертными и перцы с них получить нельзя.

**Reply by Kor-Angar** [reply]
> Мертвые пираты это Ангус (PIR_1370_Addon_Angus) и Хэнк (PIR_1371_Addon_Hank).

а с ними как? может появится? в мертвых палах в долине рудников встречаются периодически ягодки :-)

**Reply by dosinabox** [reply]
Если включить опцию, то такая вероятность появится.

**Reply by Kor-Angar** [reply]
ну так это же просто чудесно :-)

**Comment by Kor-Angar** [comment]
Бессмертные пираты? не встречал в игре.. напомни плиз

![image](https://user-images.githubusercontent.com/49514764/64246045-bdac6c00-cf14-11e9-898f-e0cf7817f316.png)
Тогда так? 21
21/6=3,5 значит 3 перца.

а зачем выпадение перца из пиратов  отключили? 

выбить из них перец нельзя сколько не сохраняйся, авторы вопрос выбивания в Готике 2 НВ пофиксили. сгенерился мир, появилось у пиратов 2 мешка, значит 2 мешка. появилось 5 ну значит 5.

мне кажется в этом плане Димус не прав, так как это не "Ягоды Гоблина" которые Пираньи могли "забыть/пропустить" этот перец они ввели именно в Ночь Ворона.

**Comment by dosinabox** [comment]
> Тогда так? 21
> 21/6=3,5 значит 3 перца.

Минус неиспользуемый PIR_1389_Addon_Tom.
20/6=3.3
3 перца

> а зачем выпадение перца из пиратов отключили?

Может, просто включить обратно? Без продажи.
Или сделать переключатель? Как должно быть по умолчанию? У пиратов или в продаже?

> Бессмертные пираты? не встречал в игре.. напомни плиз

Оказалось, что бессмертие Грега в Яркендаре снимается вместе с бессмертием Бонеса при получении бандитских доспехов. Таким образом, все пираты смертны, кроме Грега и Скипа в Хоринисе.

**Comment by Kor-Angar** [comment]
> Может, просто включить обратно? Без продажи.

я "за". Это авторское видение которое не было багом.

> Или сделать переключатель? Как должно быть по умолчанию? У пиратов или в продаже?

Это философский вопрос. на мой взгляд по умолчанию должно быть как это реализовали Пираньи: случайно появляться у пиратов. Какими мотивами руководствовался Димус я не знаю, пытался у него выяснить его логику, но он то ли не правильно мой интерес воспринял, то ли посчитал что тут и комментировать нечего :-) спроси на форуме, может кто-то прояснит этот момент.

> 3 перца

Самуэлю можно перец или два перца добавить в продажу. Мигелю и/или Фиску тоже.

Ещё одни перец был у Хуно. при его убийстве перец пропадет. тут ХЗ что сделать. как вариант сделать что бы Герой мог у него его украсть, тогда можно украсть перец, а потом его убить, без каких либо последствий.

**Comment by dosinabox** [comment]
> Ещё одни перец был у Хуно. при его убийстве перец пропадет. тут ХЗ что сделать.

Покупать)

**Comment by dosinabox** [comment]
Замечу, что хоть в обновлении и убраны перцы у пиратов, но в продаже появилось два дополнительных перца: у Сагитты в главе 4 и у Зуриса в главе 5. Так что, наверное, можно и не менять ничего сейчас. Больше нет необходимости играть в рулетку и избивать всех пиратов. Усредненное число перцев **уже** перенесено в продажу.

**Comment by Kor-Angar** [comment]
так те перцы же доступны в 4 и 5 главах. а у Пиратов со второй ;-)

**Comment by dosinabox** [comment]
А пьют сваренные зелья только в шестой)

**Comment by Kor-Angar** [comment]
Пьют то по разному🙂 в шестой опыт уже не нужен ;-)

**Comment by dosinabox** [comment]
Видимо, всё же сделаю настройку.

**Comment by Kor-Angar** [comment]
Сделай. Ибо во второй главе перец пропустить легко ;-) так же надо бы когда нибудь уже зафиксировать опыт за эликсир.

**Comment by dosinabox** [comment]
> так же надо бы когда нибудь уже зафиксировать опыт за эликсир.

На уровне шестой главы?

**Comment by Kor-Angar** [comment]
Этот топик в 2021 году уже неактуально. У тебя килотонны опыта в Минентале, поэтому опыт от эликсиров на фоне нового баланса, это просто капля в море. 0:36, 5 января 2021 г., dosinabox <notifications@github.com>:

так же надо бы когда нибудь уже зафиксировать опыт за эликсир.

На уровне шестой главы?

—You are receiving this because you modified the open/close state.Reply to this email directly, view it on GitHub, or unsubscribe.
-- Отправлено из мобильного приложения Яндекс.Почты

**Comment by Kor-Angar** [comment]
опыт за выпивания эликсира должен быть меньше чем опыт который герой получает от (Фортуно  за эликсир + Сатурас за инфу от Фортуно).

**Comment by dosinabox** [comment]
Актуально, потому что расширенный Миненталь не устанавливается по умолчанию, а в обновление в составе стим-фикса не  входит вообще.

**Comment by Kor-Angar** [comment]
Вот есть квест  у Фортуно - "лабиринты духа" + квест "инфа для Сатураса". В рамках этих квестов герой отдаёт 1 эликсир Фортуно и получает опыт от самого Фортуно + от Сатураса за инфу. В интернатах пишут что этого делать нельзя, а надо сохранить эликсир и его выпить в 5 главе -  так герой получит больше опыта.

Эликсир по сюжету герой изготавливает условно во второй главе в Ярике. Кстати квест "лабиринты духа" условная отсылка в Миненталь за рецептом и как следствие условное разделение Ярика на 2  и 3 главы (если игрок проходит Ярик во 2 главе). 

Опыт за эликсир по сюжету соответствует опыту второй(третей) главы.  Я у себя весь динамический опыт Ярика (в т. ч. за книги, телепорты и сокровища) зафиксировал на уровне второй главы. Это соседний топик https://github.com/dosinabox/g2nr_unofficial_update/issues/51 

***

> Замечу, что хоть в обновлении и убраны перцы у пиратов, но в продаже появилось два дополнительных перца: у Сагитты в главе 4 и у Зуриса в главе 5. Так что, наверное, можно и не менять ничего сейчас. Больше нет необходимости играть в рулетку и избивать всех пиратов. Усредненное число перцев уже перенесено в продажу.

Закидывания этого перца в поздние главы торговцам не решает никакой задачи :-) это как социальная политика в капитализме: пенсии проиндексировали, а инфляция их съела:
- уменьшение кол-ва перцев во второй главе, есть уменьшаешь вариативность прохождения квеста Лабиринта Духа
- перенос перца в поздние главы - есть мотивация пить  эликсир в поздних главах

***

сама игра и баланс в ней не подразумевает что игрок взял и нашёл всё перцы / коронки / ягодки или докачался до 90 и потом всё съел, выпил...
игра старая и не подразумевает 100% исследования всего, там по мере так сказать прохождения что то находится. поэтому и оружие в разных местах на тот случай если туда пошёл, там нашёл. если сюда пошёл то тут нашёл и всё дублируется по несколько раз, что бы квесты выполнялись и задания закрывались.
так же и с перцем:  есть в продаже - для тех кто покупает, есть в пиратах для тех кто НПС бъёт, в Хуно рядом тоже есть. что бы перекрыть разные аудитории. либо так либо вот так, но ты его найдёшь / отберешь / купишь. а для тех кто собрал больше - символический бонус в виде опыта при выпивания зелья героем. Попытка определить аудиторию, путём сегрегации это и есть то самое зло: бить НПС нельзя, рандом зло, нормальный игрок делает вот так, вот так было плохо (мне не нравилось), а вот так хорошо (мне нравится), вот так правильно (мне понятно), а вот так не правильно (мне не понятно).
Я вот буду следующий раз игру проходит лайтовым ментом. Главное твоих лютых имбалансных орков не забыть отключить. (вот оказывается откуда взялось слово "имба").

**Comment by Kor-Angar** [comment]
в этом кстати и проблема ТЕС - повторное прохождение, показывает линейность и скудность игрового мира. и сама игра не подразумевает одновременное продвижение во всех фракциях / гильдиях. то есть условно если герой стал архимагистром домом Телванни, то вступление в гильдию бойцов и квесты по убийству крыс выглядят уже мягко говоря очень странно.
а вот в Готике если герой стал ОнД и пришёл в монастырь то там ему не предлагают подмести пол в кельях послушников или сходить за вином :-)

**Comment by dosinabox** [comment]
Свершилось)
<img width="790" alt="Screenshot_1" src="https://user-images.githubusercontent.com/39790055/142278032-3bedd8c1-fff6-465b-8cd3-f455bde856c0.png">

**Comment by Kor-Angar** [comment]
заголовок кстати логичнее было бы сделать сбоку как сделал ElderGamer. я им обчыно не пользовался , но когда он по центру маленький его трудно воспринимать.
